site/src/pages/tutorials/thrift/blog/create-service.mdx

```diff
@@ -0,0 +1,82 @@
+---
+menuTitle: "Create a service"
+order: 1
+category: thrift
+type: step
+targetLang: java
+---
+
+# Creating a service
+
+As the first step of the tutorial, we'll create a simple service with a dummy method in Thrift and implement the service in Java.
+
+<TutorialSteps current={1} />
+
+## What you need
+
+No preparation is required for this step. Do check that you've prepared the [prerequisites](/tutorials/thrift/blog/#prerequisites).
+
+## 1. Create a thrift file
+
+Create a thrift file, `blog.thrift` in the `{project_root}/src/main/thrift` folder as follows.
+In the thrift file, let's define the `BlogService` service with the `hello()` method.
+
+```thrift filename=blog.thrift
+namespace java example.armeria.blog.thrift
+
+/**
+ * My first BlogService in Armeria!
+ */
+service BlogService {
+    string hello()
+}
+```
+
+<Tip>
+
+See [Sample service structure](/tutorials/thrift/blog#sample-service-structure) for the overall folder structure.
+
+</Tip>
+
+## 2. Compile the thrift file
+
+Compile the `blog.thrift` file to generate Java code.
+You can refer to the full [build.gradle](https://github.com/line/armeria-examples/tree/main/tutorials/thrift/build.gradle) file for generating code with [Gradle Thrift Plugin](https://github.com/jruyi/thrift-gradle-plugin).
+
+```bash
+./gradlew build
+```
+
+You'll see the generated Java code in the `{project_root}/build/generated-sources/thrift/gen-java/example/armeria/blog/thrift/` folder.
+
+## 3. Implement the service
+
+Now, let's implement the service in Java.
+
+1. Create a file, `BlogServiceImpl.java`.
+2. Declare the `BlogServiceImpl` class implementing the `BlogService` service we defined earlier in Thrift.
+  ```java filename=BlogServiceImpl.java
+  package example.armeria.server.blog.thrift;
+
+  import example.armeria.blog.thrift.BlogService;
+
+  public class BlogServiceImpl implements BlogService.AsyncIface {}
+  ```
+3. In the `BlogServiceImpl` class, override the `hello()` method, a dummy method for temporary use to test the connection between server and client.
```

Comment:
I think it would be better to state the class name and the method name separately, making it clear to understand.

I'd like to suggest to keep it as it is.
but if you don't like the sentence with too many commas, splitting the sentence could be an alternative.

```suggestion
3. In the `BlogServiceImpl` class, override the `hello()` method. This method serves as a dummy method for temporary use to test the connection between server and client.
```

Reply:
> I think it would be better to state the class name and the method name separately, making it clear to understand.
> I'd like to suggest to keep it as it is.

I see 😄    I wasn't too strong on this. I think it's fine as-is 👍